scripts/perftune.py

```diff
@@ -743,7 +749,7 @@ def __max_rx_queue_count(self, iface):
         driver_to_max_rss = {'ixgbe': 16, 'ixgbevf': 4, 'i40e': 64, 'i40evf': 16}
 
         driver_name = ''
-        ethtool_i_lines = run_read_only_command(['ethtool', '-i', iface]).splitlines()
+        ethtool_i_lines = run_ethtool(['-i', iface]).splitlines()
```

Comment:
Isn't this `splitline()` redundant as `run_ethtool` already does it?

Is the commit message out of sync with the commit itself? This looks like a pure cleanup with no functional changes, no?

Reply:
squashed refactor of ethtool and adding __get_driver_name to fix this